Source/WebCore/css/parser/CSSPropertyParser.cpp

```diff
@@ -5877,8 +5877,9 @@ bool CSSPropertyParser::consumeOverflowShorthand(bool important)
         // paged-x or paged-y, then overflow-x and overflow-y should have the same value.
         if (xValueID == CSSValueWebkitPagedX || xValueID == CSSValueWebkitPagedY)
             xValueID = CSSValueAuto;
-    } else 
+    } else {
         yValueID = m_range.consumeIncludingWhitespace().id();
```

Comment:
I think this change doesn't conform to WebKit style. https://webkit.org/code-style-guidelines/#braces-one-line